hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/SpanTypeAttributeEnricher.java

```diff
@@ -76,74 +72,6 @@
-    if (attributeMap == null) {
-      return;
-    }
-
-    // Figure out if event is entry or exit based on other span attributes
-    Boolean isEntry = null;
-    if (attributeMap.containsKey(SPAN_KIND_KEY)) {
-      String spanKindValue = attributeMap.get(SPAN_KIND_KEY).getValue();
-      if (spanKindValue.equalsIgnoreCase(SERVER_VALUE)) {
-        isEntry = true;
-      } else if (spanKindValue.equalsIgnoreCase(CLIENT_VALUE)) {
-        isEntry = false;
-      } else {
-        LOGGER.debug("Unrecognized span.kind value: {}. Event: {}.", spanKindValue, event);
-      }
-    } else if (attributeMap.containsKey(CLIENT_KEY)) {
-      String clientValue = attributeMap.get(CLIENT_KEY).getValue();
-      if (clientValue.equalsIgnoreCase("false")) {
-        isEntry = true;
-      } else if (clientValue.equalsIgnoreCase("true")) {
-        isEntry = false;
-      } else {
-        LOGGER.debug("Unrecognized Client value: {}. Event: {}.", clientValue, event);
-      }
-    } else if (attributeMap.containsKey(envoyOperationNameAttr)) {
-      String spanType = attributeMap.get(envoyOperationNameAttr).getValue();
-      if (StringUtils.equalsIgnoreCase(envoyIngressSpanValue, spanType)) {
-        isEntry = true;
-      } else if (StringUtils.equalsIgnoreCase(envoyEgressSpanValue, spanType)) {
-        isEntry = false;
-      } else {
-        LOGGER.debug("Unrecognized envoyOperationNameAttr value: {}. Event: {}.", spanType, event);
-      }
-    } else if (StringUtils.startsWith(event.getEventName(),
-        RawSpanConstants.getValue(SpanNamePrefix.SPAN_NAME_PREFIX_SENT))) {
-      // Go Opencensus instrumentation seems to have this convention Sent. prefix
-      // meaning client/exit/backend call
-      isEntry = false;
-    } else if (StringUtils.startsWith(event.getEventName(),
-        RawSpanConstants.getValue(SpanNamePrefix.SPAN_NAME_PREFIX_RECV))) {
-      isEntry = true;
-    }
-
-    // Add the new information as an enriched attribute, not raw attribute.
-    if (isEntry == null) {
-      addEnrichedAttribute(event, spanTypeAttrName, AttributeValueCreator.create(
-          Constants.getEnrichedSpanConstant(BoundaryTypeValue.BOUNDARY_TYPE_VALUE_UNSPECIFIED)));
-    } else if (isEntry) {
-      addEnrichedAttribute(event, spanTypeAttrName, AttributeValueCreator.create(
-          Constants.getEnrichedSpanConstant(BoundaryTypeValue.BOUNDARY_TYPE_VALUE_ENTRY)));
-    } else {
-      addEnrichedAttribute(event, spanTypeAttrName, AttributeValueCreator.create(
-          Constants.getEnrichedSpanConstant(BoundaryTypeValue.BOUNDARY_TYPE_VALUE_EXIT)));
-    }
-
-    // Get the protocol and name and create API entity based on the protocol.
-    Protocol protocol = getProtocolName(event);
-    addEnrichedAttribute(event, PROTOCOL_ATTR,
-        AttributeValueCreator.create(Constants.getEnrichedSpanConstant(protocol)));
-  }
-
   @Nonnull
   public static Protocol getProtocolName(Event event) {
```

Comment:
Can you please check if this method is used anywhere else? If no, can we please change it to private? Similarly, for the method `getGrpcProtocol`

Reply:
This seems to be getting used in the test class.